lib/ssoe/configuration.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/base'
+
+module SSOe
+  class Configuration < Common::Client::Configuration::SOAP
+    def service_name
+      'SSOe'
+    end
+
+    def connection
+      @connection ||= Faraday.new(base_path,
+                                  headers: base_request_headers,
+                                  request: request_options,
+                                  ssl: ssl_options) do |faraday|
+        faraday.request :soap_headers
+        faraday.response :soap_parser
+        faraday.adapter Faraday.default_adapter
+      end
+    end
+
+    private
+
+    def ssl_options
+      raise 'SSL options not defined' unless ssl_cert && ssl_key
```

Comment:
Can prob remove this line now, shouldn't be able to happen since ssl_cert and ssl_key will cause errors now